pkg/kubelet/server/stats/summary.go

```diff
@@ -115,6 +115,8 @@ func (sb *summaryBuilder) build() (*stats.Summary, error) {
 		return nil, fmt.Errorf("Missing stats for root container")
 	}
 
+	nodeFsInodesUsed := *sb.rootFsInfo.Inodes - *sb.rootFsInfo.InodesFree
```

Comment:
you will need to check for nil on both values.  for imagefs, it will be nil for both inodes and inodesfree values.